pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -54,6 +63,59 @@ func formatReason(reasons []algorithm.PredicateFailureReason) string {
 	return strings.Join(reasonStrings, ", ")
 }
 
+type predicateEnable struct {
+	memoryPressureEnable bool
+	diskPressureEnable   bool
+	pidPressureEnable    bool
+}
+
+func initPredicateEnable() predicateEnable {
+	predicate := predicateEnable{
+		memoryPressureEnable: false,
+		diskPressureEnable:   false,
+		pidPressureEnable:    false,
+	}
+	return predicate
+}
+
+func enablePredicate(args framework.Arguments) predicateEnable {
+
+	/*
+		   User Should give predicatesEnable in this format(predicate.NodeConditionEnable, predicate.NodeUnschedulableEnable, predicate.PodTolerationEnable,
+		   predicate.MemoryPressureEnable, predicate.DiskPressureEnable, predicate.PIDPressureEnable, predicate.PodAffinityEnable).
```

Comment:
There is no `predicate.PodAffinityEnable`.

Reply:
Add it might be better to move them to the doc?

Reply:
First I was thinking whether PodAffinity Predicate should be default? Then I changed it to be default predicates, so in Doc I would have missed, will edit that.

Reply:
Doc has also been added for users to give configuration accordingly